src/zenml/zen_server/rbac/utils.py

```diff
@@ -501,7 +501,7 @@ def get_resource_type_for_model(
         PipelineResponse: ResourceType.PIPELINE,
         CodeRepositoryResponse: ResourceType.CODE_REPOSITORY,
         SecretResponseModel: ResourceType.SECRET,
-        ModelResponseModel: ResourceType.MODEL,
+        ModelResponse: ResourceType.MODEL,
```

Comment:
Something I missed in my RBAC PR and not currently affecting anything in the logic, but could you also add `ModelVersionResponse: ResourceType.MODEL_VERSION` here for completeness? 🙏

Reply:
Added it!